src/main/java/models/League.java

```diff
@@ -0,0 +1,46 @@
+package models;
+
+import java.util.LinkedList;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Queue;
+
+public class League {
+
+    private Queue<Team> initialTeams;
+    private HashMap<Integer, ArrayList<Team>> winners;
```

Comment:
I think the `Integer` here is the round number but am not 100% certain. Possibly clear this up with a comment for faster understandability!